src/components/Board/Board.component.js

```diff
@@ -298,6 +378,75 @@ export class Board extends Component {
             'is-locked': this.props.isLocked
           })}
         >
+          {isLocked && (
+            <Joyride
+              callback={data => {
+                const { status } = data;
+                if ([STATUS.FINISHED, STATUS.SKIPPED].includes(status)) {
+                  this.setState({ runLiveUnlock: false });
+                }
+              }}
+              steps={firststep}
+              continuous={true}
+              showSkipButton={true}
+              showProgress={true}
+              run={runLiveUnlock}
+              styles={{
+                options: {
+                  // modal arrow and background color
+                  arrowColor: '#eee',
+                  backgroundColor: '#eee',
+                  //button color
+                  primaryColor: '#aa00ff',
+                  //text color
+                  textColor: '#333',
+
+                  //width of modal
+                  width: 500,
+                  //zindex of modal
+                  zIndex: 1000
+                }
+              }}
+              locale={{
+                last: 'End tour',
+                skip: 'Close tour'
+              }}
+            />
+          )}
+          {!isLocked && (
+            <Joyride
+              callback={data => {
+                const { status } = data;
+                if ([STATUS.FINISHED, STATUS.SKIPPED].includes(status)) {
+                  this.setState({ runLiveHelp: false });
+                }
+              }}
+              steps={steps}
+              continuous={true}
+              showSkipButton={true}
+              showProgress={true}
+              run={runLiveHelp}
+              styles={{
+                options: {
+                  // modal arrow and background color
+                  arrowColor: '#eee',
+                  backgroundColor: '#eee',
+                  //button color
+                  primaryColor: '#aa00ff',
+                  //text color
+                  textColor: '#333',
+                  //width of modal
+                  width: 500,
+                  //zindex of modal
+                  zIndex: 1000
+                }
+              }}
+              locale={{
+                last: 'End tour',
```

Comment:
Missing translated string "End tour", "Close tour"

Reply:
Added to board.messages.js & cboard.json!